src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$green-color: rgb(123, 170, 123);
+$blue-color: rgb(96, 139, 191);
```

Comment:
you make only two color variables, but in your project you use four color I think, so maybe it would be better to add all colors to variables?

Reply:
yeah, that makes sense)